lms/templates/learner_dashboard/certificate.underscore

```diff
@@ -1,7 +1,7 @@
 <div class="certificate-container">
-    <p class="title"><%- gettext('XSeries Program Certificates') %>:</p>
+    <h2 class="hd-6"><%- gettext('XSeries Program Certificates') %>:</h2>
```

Comment:
Yeah, it would be nice if there were default styles for headings but I fear that could lead to issues with people using the (semantically) incorrect header for the page to get it to look like the design they want. I guess ideally we would have default styles that map to the Pattern Library hd-1, hd-2, hd-3, etc. with the ability to override (if the DOM dictates that it should be an h3 but you want it to look like an h2). But I think this may be out of scope of this PR.

I agree that the naming of the styles is too tightly related to the DOM elements, but can't really suggest a better name for 6 levels of headings :)